### DISCUSSION TITLE
(python3-streams) Please don't forget python310!

### DISCUSSION THREAD

**Discussion by karig** [chocolatey-community/chocolatey-packages#2084] [Ideas]
python3-streams.json:
```
{
    "3.11":  "3.11.1",
    "3.12":  "3.12.0-a2"
}
```
→ ```"3.10": "3.10.9",```

**Comment by AdmiringWorm** [comment]
I am sorry that this one has gone unnoticed. The python 3.10 streams were never forgotten. However, until we had 3.11 approved, we wanted to limit what was being pushed in case changes were needed.

As the streams have been enabled again, this reminder is no longer needed, and I'll close this discussion.
